en/docs/deploy/security/use-symmetric-encryption.md

```diff
@@ -1,6 +1,6 @@
 # Configurations Related to Symmetric Key Encryption
 
-This section explains the configurations related to [symmetric key encryption](../../../deploy/security/symmetric-overview). From WSO2 Identity Server 5.11.0 onwards, symmetric encryption will be used by default. You can switch to [asymmetric key encryption](../../../deploy/security/use-asymmetric-encryption) if required.
+This section explains the configurations related to [symmetric key encryption](../../../deploy/security/symmetric-overview). This version of WSO2 Identity Server uses symmetric encryption by default. You can switch to [asymmetric key encryption](../../../deploy/security/use-asymmetric-encryption) if required.
```

Comment:
`This version of WSO2 Identity Server uses symmetric encryption by default.` -> contradicting with https://github.com/wso2/docs-is/pull/2649/files#diff-87b30ea095c7c836e6e9202cfaba9d93ed0bf7545260261857f13dfbe0705d17R5